api-report/merge-tree.api.md

```diff
@@ -285,23 +279,17 @@ export function createAnnotateRangeOp(start: number, end: number, props: Propert
 // @public (undocumented)
 export function createGroupOp(...ops: IMergeTreeDeltaOp[]): IMergeTreeGroupMsg;
 
-// @public (undocumented)
-export function createInsertFromRegisterOp(pos: number, register: string): IMergeTreeInsertMsg;
```

Comment:
I'm naïve about how ops are represented in code. Does this imply we also need to make a change elsewhere to remove the "op type"?

Reply:
the op types are overloaded. this op is still used for regular inserts. just removing the register part

Reply:
this was a good question btw. it made me realize we should blow up the op processing if we detect this type, rather than ignore, which could lead to corruption: https://github.com/microsoft/FluidFramework/pull/8139/files#r743823290

Reply:
Tony got to it before me, but yeah:

The "insert" and "remove" ops were pulling double-duty as "plain insert" and "register insert" ops, sort of.  This change removes the `register` member from them, such that there's only "plain insert" remaining.